proto/sedachain/core/v1/core.proto

```diff
@@ -112,16 +112,18 @@ enum DataRequestStatus {
 message RevealBody {
   // Hex-encoded ID of the data request
   string dr_i_d = 1 [ (gogoproto.jsontag) = "dr_id" ];
+  // Hex-encoded public key of the executor who has revealed this.
+  string executor = 2;
```

Comment:
This is an addition to the `RevealBody` compared to what we used to have. Why is this necessary? It also means we have to update the hash, and reveal body size limits since it's another field...

Reply:
Maybe it's ok because we overwrite it and ignore any value given https://github.com/sedaprotocol/seda-chain/blob/96d991c7cd030cfbeefc32ecf2b26c6a3a2ea0fb/x/core/keeper/msg_server_dr.go#L349. @mariocao can you double-check to make sure I'm not missing anything security wise?

It would still affect gas costs, and RPC possibly performance.